cmd/policygen/main.go

```diff
@@ -62,6 +76,8 @@ func main() {
 
 	rn := &runner.Default{Quiet: true}
 	if err := policygen.Run(context.Background(), rn, args); err != nil {
-		log.Fatalf("Failed to generate policies: %v", err)
+		return fmt.Errorf("Failed to generate policies: %v", err)
```

Comment:
lower case

Reply:
It is now. You might be looking at an older copy.